file-formats/ddls/type/zif_aff_ddls_v1.intf.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf.json",
```

Comment:
has to refer to the cds schema not the intf schema

Reply:
Not, I think this is correct, as it is the serialized interface for DDLS